app/components/Form/ProjectQuarterlyReportFormSummary.tsx

```diff
@@ -50,6 +47,9 @@ const ProjectQuarterlyReportFormSummary: React.FC<Props> = ({
                 formChangeByPreviousFormChangeId {
                   newFormData
                 }
+                formByJsonSchemaName {
```

Comment:
Is there any difference getting the json schema this way vs `formBySlug`?

Reply:
I don't think so, there are often different ways to get the same piece of info (but if you have concerns, let me know, I'm still new to graphql)